### DISCUSSION TITLE
Question: how can I use different config according to different env param?

### DISCUSSION THREAD

**Discussion by 349989153** [postcss/postcss#1820] [General]
I have a vue project and I'm trying to make it‘s  CSS style transform into REM in production but not in development. So I tried like: 
`package.json`:
```json
"scripts": {
  "serve:rem": "CSS_ENV=rem vue-cli-service serve",
}
```

`postcss.config.js`:
```js
module.exports = (ctx) => {
  console.log('ctx', ctx);
  return ({
    plugins: [
      ctx.env.CSS_ENV === 'rem' // doesn't work, value of ctx.env is either 'production' or 'development' only
        ? require('postcss-pxtorem')({
        rootValue: 16,
        propList: ["*"],
        selectorBlackList: [".am-", ".disrem-"],
      }) : null
    ]
  });
}

```

**Comment by 349989153** [comment]
Or maybe I can create 2 config file like: 
`postcss.config.js`
`postcss.config.dev.js`

But how to specify config file in `scripts` in `package.json`?

**Comment by ai** [comment]
You should use `process.env.CSS_ENV`

**Reply by 349989153** [reply]
Oh, yeah, `process.env.CSS_ENV` did the work, I hadn't even thought about that， HAHA
